hypothesis-python/tests/cover/test_regressions.py

```diff
@@ -101,3 +104,19 @@ def test_false_is_false(params):
 
     with pytest.raises(ValueError):
         test_false_is_false()
+
+
+@given(st.integers())
+def random_func(x):
+    random.random()
+
+
+def test_prng_state_unpolluted_by_given():
+    initial = random.getstate()  # so we can clean up after *this* test
+    random.seed(os.urandom(10))  # ensure unconnected to future PRNG bugs
+
+    before = random.getstate()
+    random_func()
+    assert random.getstate() == before, 'shoudl restore PRNG to previous state'
```

Comment:
sp: “shoudl”

Reply:
Ugh, that's what I get for late-night testing...  will fix.